src/main/java/serialization/ReadObject.java

```diff
@@ -0,0 +1,27 @@
+package main.java.serialization;
+
+import java.io.*;
+
+public class ReadObject {
+    public static void main(String[] args) {
+
+        try (FileInputStream fis = new FileInputStream("people.bin");
+             ObjectInputStream ois = new ObjectInputStream(fis)) {
+
+            Person person1 = (Person) ois.readObject();
+            Person person2 = (Person) ois.readObject();
+
+            System.out.println(person1);
+            System.out.println(person2);
+        } catch (FileNotFoundException e) {
+            System.out.println("File not found");
+            e.printStackTrace();
+        } catch (ClassNotFoundException e) {
+            System.out.println("Class not found");
+            e.printStackTrace();
+        } catch (IOException e) {
+            System.out.println("IO exception");
+            e.printStackTrace();
```

Comment:
What is the difference between `printStackTrace` and `getMessage`?

Reply:
![photo_2023-10-10_15-37-41](https://github.com/Buga1chukY/JavaPractice/assets/123409653/30905aa8-8a38-4e46-866c-1f66dd3520a9)  Method printStackTrace give us more information about error, we recive the stack call when exception was thrown.


![photo_2023-10-10_15-37-33](https://github.com/Buga1chukY/JavaPractice/assets/123409653/16e628b8-099a-430a-9152-1cd0642b0ed0) GetMessage gie us reason of the error. We can understand why it was happend, but we don't know in what time if code it happend.